engine/src/field.h

```diff
@@ -239,7 +240,8 @@ class MCField : public MCControl, public MCMixinObjectHandle<MCField>
 	MCStringRef label;
     MCTextDirection text_direction;
     MCInterfaceFieldCursorMovement cursor_movement;
-    
+    intenum_t keyboard_type;
```

Comment:
Hmm... regarding the bitfield I can't help thinking it's risky for an enum because it would be easy to change the enum and not see that the variable is a bitfield. I realise there's a low risk of having more than 16 values in this instance I'm just wondering if we might want to not make this a thing we do...

Reply:
Hence why I said we needed roundtrip tests for the property values (i.e. repeat for each item tValue in "..."; set the  returnKeyType of field 1 to tValue; TestAssert ..., tValue is the returnKeyType of field 1; end repeat).

Reply:
aha... woops forgot to add those will do